ui/packages/consul-ui/app/components/topology-metrics/card/index.hbs

```diff
@@ -6,10 +6,11 @@
 </a>
 {{else}}
 <a class="topology-metrics-card"
+  data-test-topology-metrics-card
```

Comment:
I noticed we've added this on this anchor, but not the anchor above here (that you can't see in the GH diff). If you are adding it to one, should it be added to both? I'm not sure if you can, but if you could only have one anchor element and then construct its contents with conditionals, that would be better I think (this is similar to another comment I had on another PR recently)

Reply:
yea, I'll add it to both anchors.